android-client-sdk/build.gradle

```diff
@@ -73,6 +73,7 @@ ext {
     mockk_version = '1.12.4'
     hamcrest_version = "2.2"
     okhttp_version = "4.9.3"
+    ld_eventsource_version = "2.7.1"
```

Comment:
should this dependency be here? it seems like you're just bundling the whole library in the SDK?

Reply:
no its not, I forgot to remove it